src/Umbraco.Web.UI.Client/src/packages/media/dropzone/components/input-dropzone/input-dropzone.element.ts

```diff
@@ -60,6 +60,12 @@ export class UmbInputDropzoneElement extends UmbFormControlMixin<UmbUploadableIt
 	@property({ type: Boolean })
 	multiple: boolean = false;
 
+	/**
+	 * @description The style the dropzone with a border.
```

Comment:
```suggestion
	 * Style the dropzone with a border.
	 * @description This is useful if you want to display the dropzone directly rather than as a part of a separate component.
```